assets/svg/graphics/cta-graph-arrow-up.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="6" height="6" viewBox="0 0 6 6" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@techanvil As mentioned in my earlier comment, let's move it to the `assets/svg/icons` folder and rename it to `arrow-up.svg`

Reply:
That's done :+1: